src/main/java/frc/robot/RobotContainer.java

```diff
@@ -4,60 +4,42 @@
-  // The robot's subsystems and commands are defined here...
-  private final ExampleSubsystem m_exampleSubsystem = new ExampleSubsystem();
-
-  // Replace with CommandPS4Controller or CommandJoystick if needed
-  private final CommandXboxController m_driverController =
-      new CommandXboxController(OperatorConstants.kDriverControllerPort);
-
-  /** The container for the robot. Contains subsystems, OI devices, and commands. */
-  public RobotContainer() {
-    // Configure the trigger bindings
-    configureBindings();
-  }
-
-  /**
-   * Use this method to define your trigger->command mappings. Triggers can be created via the
-   * {@link Trigger#Trigger(java.util.function.BooleanSupplier)} constructor with an arbitrary
-   * predicate, or via the named factories in {@link
-   * edu.wpi.first.wpilibj2.command.button.CommandGenericHID}'s subclasses for {@link
-   * CommandXboxController Xbox}/{@link edu.wpi.first.wpilibj2.command.button.CommandPS4Controller
-   * PS4} controllers or {@link edu.wpi.first.wpilibj2.command.button.CommandJoystick Flight
-   * joysticks}.
-   */
-  private void configureBindings() {
-    // Schedule `ExampleCommand` when `exampleCondition` changes to `true`
-    new Trigger(m_exampleSubsystem::exampleCondition)
-        .onTrue(new ExampleCommand(m_exampleSubsystem));
-
-    // Schedule `exampleMethodCommand` when the Xbox controller's B button is pressed,
-    // cancelling on release.
-    m_driverController.b().whileTrue(m_exampleSubsystem.exampleMethodCommand());
-  }
-
-  /**
-   * Use this to pass the autonomous command to the main {@link Robot} class.
-   *
-   * @return the command to run in autonomous
-   */
-  public Command getAutonomousCommand() {
-    // An example command will be run in autonomous
-    return Autos.exampleAuto(m_exampleSubsystem);
-  }
+
+    private final DriveSubsystem driveBase = new DriveSubsystem();
+    private final XboxController driveController = new XboxController(Constants.XBOXCONTORLLER.hidport());
+
+    public RobotContainer() {
+        configureBindings();
+        driveBase.setDefaultCommand(driveFieldOrientedAngularVelocity);
+    }
+
+    SwerveInputStream driveAngularVelocity = SwerveInputStream.of(
+                    driveBase.getSwerveDrive(),
+                    () -> driveController.getLeftY() * -1,
+                    () -> driveController.getLeftX() * -1)
+            .withControllerRotationAxis(driveController::getRightX)
+            .deadband(Constants.DEADBAND)
+            .scaleTranslation(0.8)
+            .allianceRelativeControl(true);
+
+    SwerveInputStream driveDirectAngle = driveAngularVelocity
+            .copy()
+            .withControllerHeadingAxis(driveController::getRightX, driveController::getRightY)
+            .headingWhile(true);
+
+    Command driveFieldOrientedDirectAngle = driveBase.driveFieldOriented(driveDirectAngle);
+
+    Command driveFieldOrientedAngularVelocity = driveBase.driveFieldOriented(driveAngularVelocity);
```

Comment:
nit: it's a bit confusing declaring these variables lower down in the file after they're used. It's syntactically valid, but I would prefer if you could move these lines above the constructor for increased legibility